### ISSUE TITLE
Genius SW-HF5.1 5050 V2 (working config)

### ISSUE THREAD

**Issue by sukhoi191** [irplus-remote/irplus-codes.github.io#572]
Layout based on existing Genius SW-HF5.1 6000 (same buttons, different codes).

[GENIUS - SW-HF5.1 5050 V2.zip](https://github.com/irplus-remote/irplus-codes.github.io/files/8827994/GENIUS.-.SW-HF5.1.5050.V2.zip)

**Comment by flxrin1337** [comment]
Thanks for posting, sadly, it doesn't work. :(

**Comment by irplus-remote** [comment]
@flxrin1337 It should work, as they were tested, however maybe it is only for 6000 series? Are you using irplus WAVE or irplus with integrated ir blaster? Which Genius exactly do you have?

**Comment by flxrin1337** [comment]
> @flxrin1337 It should work, as they were tested, however maybe it is only for 6000 series? Are you using irplus WAVE or irplus with integrated ir blaster? Which Genius exactly do you have?

I do have the Genius SW-HF 5.1 5050 V2, as it's written on the label on the back on the head unit. I use irplus - Infrared Remote (not the wave or LAN version), and as hardware, I used a Samsung Galaxy S4 Value Edition with built-in IR blaster, Android 5.0 official software, and a Samsung Galaxy S5, also with built-in IR blaster, using a custom software by Lineage, LineageOS v18.1 based on Android 11. The app works perfectly fine on both of the devices, testing them with a Samsung TV, but for the head unit, no luck.

**Comment by sukhoi191** [comment]
![5050_v2_2](https://user-images.githubusercontent.com/6483306/179418002-43dcb03d-dd2e-4060-83f5-c12d62fae0fd.jpg)

Do your speakers look like this?

**Comment by flxrin1337** [comment]
> ![5050_v2_2](https://user-images.githubusercontent.com/6483306/179418002-43dcb03d-dd2e-4060-83f5-c12d62fae0fd.jpg)
> 
> 
> 
> @flxrin1337
> 
> Do your speakers look like this?
> 
> 
> 
> I just double-checked and it seems I provided a correct model. My speakers look like those in the image. Here's a photo of the label:
> 
> 
> 
> ![IMG20220717194146](https://user-images.githubusercontent.com/6483306/179418100-68be3f6f-60d2-4a92-9c3d-a2157ff9b941.jpg)

Yup, they do. ![image](https://user-images.githubusercontent.com/37021829/179419439-b60540ff-8c86-428a-9a3d-5ee3c98f64c3.jpeg)

Idk why it's not working..
